src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psm1

```diff
@@ -48,3 +48,35 @@ function Create-AzApiManagementApiOperation {
 }
 
 Export-ModuleMember -Function Create-AzApiManagementApiOperation
+
+<#
+ .Synopsis
+  Import a policy to an product in Azure API Management.
+
+ .Description
+  Import a policy to an API scope in a Azure API Management service.
+
+ .Parameter ResourceGroupName
+  The resource group containing the API Management service.
+
+ .Parameter ServiceName
+  The name of the API Management service located in Azure.
+
+ .Parameter ProductId
+  The ID to identify the product running in API Management.
```

Comment:
It's not really something that is running in APIM, it's more of a identifier/shared configuration into which API's can be added.
```suggestion
  The ID to identify the product in API Management.
```